### ISSUE TITLE
Create icon for standalone ROITool

### ISSUE THREAD

**Issue by stelfrich** [mutterer/figurej#7]
Once #3 is in place, we will need an icon for the new `ROITool`.

**Comment by mutterer** [comment]
Makes sense, and maybe an image activated listener could take care of switching to the figure tool when the figure is activated and to the ROITool when 'open image' button is used.
